### PR TITLE
[INLONG-3225][Manager] Resolves multiple IPs when querying a DataProxy cluster

### LINKED ISSUE #3225: [Improve][Manager] Resolves multiple IPs when querying a DataProxy cluster
### Description

Resolves multiple IPs when querying a DataProxy cluster.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Title Name: [INLONG-3225][Manager] Resolves multiple IPs when querying a DataProxy cluster

Fixes #3225 

### Motivation

Resolves multiple IPs when querying a DataProxy cluster.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [X] Make sure that the change passes the CI checks.

This change added tests and can be verified as follows:

  - `org.apache.inlong.manager.service.core.impl.InlongClusterServiceTest`

### Documentation

  - Does this pull request introduces a new feature? (no)

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

File: inlong-manager/manager-test/src/main/resources/sql/apache_inlong_manager.sql [hunk 1]
```diff
@@ -151,7 +151,7 @@ CREATE TABLE `third_party_cluster`
     `id`          int(11)      NOT NULL AUTO_INCREMENT COMMENT 'Incremental primary key',
     `name`        varchar(128) NOT NULL COMMENT 'Cluster name',
     `type`        varchar(32)  NOT NULL COMMENT 'Cluster type, including TUBE, PULSAR, etc.',
-    `ip`          varchar(64)  NOT NULL COMMENT 'Cluster IP',
+    `ip`          text         NOT NULL COMMENT 'Cluster IP',
```
Comment (baomingyu) [inline inlong-manager/manager-test/src/main/resources/sql/apache_inlong_manager.sql:154]: please, change this line comment, and give one example.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ThirdPartyClusterServiceImpl.java [hunk 2]
```diff
@@ -177,28 +177,59 @@ public Boolean delete(Integer id, String operator) {
...
 
     @Override
-    public List<DataProxyResponse> getIpList(DataProxyRequest request) {
-        LOGGER.debug("begin to list data proxy by request={}", request);
-        List<ThirdPartyClusterEntity> entityList = thirdPartyClusterMapper.selectByType(Constant.CLUSTER_DATA_PROXY);
-        if (entityList == null || entityList.isEmpty()) {
-            LOGGER.warn("success to list data proxy, but not found anything for request={}", request);
+    public List<DataProxyResponse> getIpList(String clusterName) {
+        LOGGER.debug("begin to list data proxy by clusterName={}", clusterName);
+        ThirdPartyClusterEntity entity = thirdPartyClusterMapper.selectByName(clusterName);
```
Comment (baomingyu) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ThirdPartyClusterServiceImpl.java:182]: this change need client must provide clusterName as parameter ?

**Review by baomingyu — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-test/src/main/resources/sql/apache_inlong_manager.sql (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-test/src/main/resources/sql/apache_inlong_manager.sql:154]: OK, I changed it, if the cluster name was empty, query data proxy cluster by type and select the first one to return.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ThirdPartyClusterServiceImpl.java (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ThirdPartyClusterServiceImpl.java:182]: OK, I changed it, if the cluster name was empty, query data proxy cluster by type and select the first one to return.

**Review by baomingyu — APPROVED** [review]
+1

**Pull request merged**